flink-tests/src/test/java/org/apache/flink/test/state/operator/restore/keyed/KeyedJob.java

```diff
@@ -98,10 +98,9 @@ public static void main(String[] args) throws Exception {
 			.map(new StatefulStringStoringMap(mode, "first"))
 			.setParallelism(4);
 
-		// TODO: re-enable this when generating the actual 1.2 savepoint
-		//if (mode == ExecutionMode.MIGRATE || mode == ExecutionMode.RESTORE) {
-		map.uid("first");
-		//}
+		if (mode == ExecutionMode.MIGRATE || mode == ExecutionMode.RESTORE) {
```

Comment:
Yes, thank you very much, now I get it :)

I think since this code is already on the master it's a bit too late change a commit message there :(